### ISSUE TITLE
Filter by column value

### ISSUE THREAD

**Issue by timboven** [stevenrskelton/sortable-table#27]
This table-component has a lot of handy features that make it usable in multiple places but adding a filter-option that e.g. shows an input-box in each column header that can be used to filter the rows of the table would make it even handier.

This would filter the rows where the value of that column doesn't start(/contains?) the given filter-value.
Users can use this to get that part out of the resultset that interests them the most.

Similar components (like rich:datatable of richfaces) provide similar features.

**Comment by stevenrskelton** [comment]
Agreed, filtering will be nice.  v0.11.0 introduced some basic filtering, but to do this properly a couple things still need to be implemented.

**1**: Column types, so that applicable filters display for specific columns.  
**2**: Value pickers for each comparison type; a date/calendar picker needs to be selected
**3**: Remote/server-side datasources need to updated to support it.
**4**: Applied filters need to be visible graphically
**5**: Filters need to be theme-able, or definable using a `template`

**Comment by LM450N** [comment]
I would like to have a `paper-input` element and keeping the rows who have at least one value on a column who starts with the given value.

Is it possible to do this currently (by the `sortable-table`) ?

If not, is it a good idea to used a copy of data and update it in according to the filter field ?
